src/foam/nanos/se/cronjobs.jrl

```diff
@@ -0,0 +1,76 @@
+
+p({
+  "class": "foam.nanos.cron.Cron",
+  "id": "systemevent_cron",
+  "description": "cron to activate/deactivate a systemevent",
+  "clusterable": true,
+  "enabled": true,
+  "schedule": {
+    "class":"foam.nanos.cron.IntervalSchedule",
+    "duration":{
+      "class":"foam.nanos.cron.TimeHMS",
+      "minute":1
+    }
+  },
+  "code": """
+    import foam.nanos.se.SystemEvent;
+    import foam.nanos.logger.Loggers;
+    import foam.dao.DAO;
+    import foam.dao.ArraySink;
+    import static foam.mlang.MLang.*;
+    import java.util.Calendar;
+    import java.util.List;
+    import java.util.TimeZone;
+
+    var currTime = Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime();
+
+    var systemEventDAO = x.get("systemEventDAO");
+    List<SystemEvent> activate = ((ArraySink) systemEventDAO
+      .where(
+        AND(
+          EQ(SystemEvent.ENABLED, true),
+          EQ(SystemEvent.ACTIVE, false),
+          GT(SystemEvent.END_TIME, SystemEvent.START_TIME),
```

Comment:
endtime is optional.  for a systemevent that an alarm may re-enable, it would set the starttime to now, but can't set the endtime. so the endtime may be less than that start time.